src/site/xdoc/server/config-users.xml

```diff
@@ -41,8 +41,11 @@
         <li>username should not be longer than 255 characters</li>
         <li>username can not contain '/'</li>
         <li>username can not contain multiple domain delimiter('@')</li>
-        <li>A username can have only local part. E.g.'myUser'</li>
-        <li>A username can have domain part, and the domain part should be concatenated after a domain delimiter('@'). E.g. 'myuser@james.org'</li>
```

Comment:
When virtualHosting is enabled, a username should have domain part, and the domain part should be concatenated after a domain delimiter('@'). E.g. 'myuser@james.org